test/fail_compilation/isreturnonstack.d

```diff
@@ -0,0 +1,12 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/isreturnonstack.d(11): Error: argument to `__traits(isReturnOnStack, int)` is not a function
+fail_compilation/isreturnonstack.d(12): Error: expected 1 arguments for `isReturnOnStack` but had 2
```

Comment:
Isn't it a bit inconsistent to have `__traits(isReturnOnStack ...` in one error message and `isReturnOnStack` in another?

Reply:
It's consistent with all the other error messages put out by traits.d. If you want to redo all that, that's ok, but it would be a separate PR.

Reply:
Didn't know that.